lib/logatron/logatron.rb

```diff
@@ -50,12 +50,28 @@ def debug(msg)
       logger.debug(msg)
     end
 
+    # Deprecated; use request_info/parse_request_info for transportation-layer neutral methods.
     def http_headers
       {
-        'X-Ascent-Log-Id' => msg_id
+        'X-Ascent-Log-Id' => msg_id,
+        'X-Ascent-Site' => site
       }
     end
 
+    def request_info
```

Comment:
I'm wondering if there's a better name. Is this a "context"? Is "request" referring to the original client request? In that case the "request" _is_ the context for a number of internal requests (conversion, processing, etc.)